lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -297,52 +297,81 @@ describe('NxCombobox', function() {
       expect(inputElement).toHaveAttribute('aria-activedescendant', optionBtns[0].id);
     });
 
-    it('places the editing cursor at the begining of the input field when Home key is pressed', async function() {
-      const onChange = jest.fn(),
-          { getByRole } = quickRender({ value: 'f', onChange }),
-          inputElement = getByRole('combobox');
-
-      inputElement.focus();
-      await userEvent.keyboard('[Home]o');
-      expect(onChange).toBeCalledWith('of');
-    });
-
-    it('places the editing cursor at the end of the input field when End key is pressed', async function() {
-      const onChange = jest.fn(),
-          { getByRole} = quickRender({ value: 'f', onChange }),
-          inputElement = getByRole('combobox');
-
-      inputElement.focus();
-      await userEvent.keyboard('[Home][End]o');
-      expect(onChange).toBeCalledWith('fo');
-    });
+    it('places the editing cursor at the begining of the input field when Home key is pressed with focus in combobox',
+        async function() {
+          const onChange = jest.fn(),
+              { getByRole } = quickRender({ value: 'f', onChange }),
+              inputElement = getByRole('combobox');
+
+          inputElement.focus();
+          await userEvent.keyboard('[Home]o');
+          expect(onChange).toBeCalledWith('of');
+        });
+
+    it('places the editing cursor at the end of the input field when End key is pressed with focus in combobox',
+        async function() {
+          const onChange = jest.fn(),
+              { getByRole} = quickRender({ value: 'f', onChange }),
+              inputElement = getByRole('combobox');
+
+          inputElement.focus();
+          await userEvent.keyboard('[Home][End]o');
+          expect(onChange).toBeCalledWith('fo');
+        });
+
+    it('places the editing cursor at the begining of the input field when Home key is pressed with focus in dropdown',
+        async function() {
+          const onChange = jest.fn(),
+              { getByRole } = quickRender({ matches: [{ id: '1', displayName: 'Foo' }], value: 'f', onChange }),
+              inputElement = getByRole('combobox'),
+              optionBtn = getByRole('option', { hidden: true });
+
+          inputElement.focus();
+          await userEvent.keyboard('[ArrowDown]');
+          expect(optionBtn).toHaveAttribute('aria-selected', 'true');
+          await userEvent.keyboard('[Home]o');
+          expect(onChange).toBeCalledWith('of');
```

Comment:
Should probably test that `optionBtn` no longer has `aria-selected` set

Reply:
Updated here 238f37f503fd02026e52011818f0433200baeae8. Thanks.